### PR TITLE
Add play button to simulator.

### PR DESCRIPTION
**Pull request by microbit-robert**

### CODE REVIEW

**Comment by microbit-robert** [comment]
Items for discussion:

- Exact button icon and styling. Temporary version currently applied.
- Should the play button cover the whole board as it does currently? It's a much bigger click target, but the tab focus indicator is less clear.

File: src/board/ui.ts [hunk 1]
```diff
@@ -18,12 +19,32 @@ export function createBoard(
...
+  playButton.style.display = "flex";
+  playButton.style.justifyContent = "center";
+  playButton.style.alignItems = "center";
+  playButton.style.width = "100%";
+  playButton.style.height = "100%";
+  playButton.style.top = "0";
+  playButton.style.left = "0";
+  playButton.style.backgroundColor = "transparent";
+  playButton.style.border = "none";
+  playButton.style.cursor = "pointer";
```
Comment (microbit-matt-hillsdon) [inline src/board/ui.ts]: It might be nicer to define a class in simulator.html ?

Comment (microbit-robert) [reply src/board/ui.ts]: My brain was stuck in 'everything in JavaScript' mode... [`72b1746`](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/23/commits/72b174643bcaed00b861fb42fb9d32eced1cd41d).

File: src/board/ui.ts [hunk 2]
```diff
@@ -18,12 +19,34 @@ export function createBoard(
...
+  stoppedOverlay.appendChild(playButton);
+  return stoppedOverlay;
+}
+
+function getButtonColor() {
+  const params = new URLSearchParams(window.location.search);
+  if (params.has("color")) {
+    return params.get("color");
+  }
 }
```
Comment (microbit-matt-hillsdon) [inline src/board/ui.ts]: Do we need the `has` check? It returns null if missing which might be find for our purposes.

Even if intentional, returning undefined like this by falling off the end of the function looks like a bug. If we want that behaviour then let's do it explicitly and type the return value.

File: src/demo.html [hunk 3]
```diff
@@ -67,7 +67,7 @@ <h1>MicroPython-micro:bit simulator example embedding</h1>
         <div class="simulator column">
           <iframe
             id="simulator"
-            src="simulator.html"
+            src="simulator.html?color=%236c4bc1"
```
Comment (microbit-matt-hillsdon) [inline src/demo.html]: Let's not use the micro:bit brand colour in this demo. blue? Or whatever 😄 

Might be nice to document it in README.md too.

**Review by microbit-matt-hillsdon — CHANGES_REQUESTED** [review]
Some tweaks but looks good.

File: src/board/ui.ts (hunk 2 above)
Comment (microbit-robert) [reply src/board/ui.ts]: [`c01dc04`](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/23/commits/c01dc04b348978a971f469467a68a8d3394dc0b3).

File: src/demo.html (hunk 3 above)
Comment (microbit-robert) [reply src/demo.html]: [`c577b9f`](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/23/commits/c577b9f70b00036f292e34ed99f130d058d8540b) & README.md updated here [`546d5c7`](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/23/commits/546d5c7b6bcb876d35765bcc8ea4015463b2bc66).

**Review by microbit-matt-hillsdon — APPROVED** [review]
Change look good, will merge once it builds.

**Pull request merged**